docs/admin/eventing/broker-configuration.md

```diff
@@ -13,7 +13,7 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
 data:
-  # Configuration for defaulting Channels that do not specify CRD implementations.
+  # Configuration for defaulting brokers that do not specify a spec.config or broker class.
```

Comment:
```suggestion
  # Configuration for default Brokers that do not specify a spec.config or Broker class.
```

Reply:
The comment is correct. This is used for defaulting _all_ Brokers, not for _the default Broker_.